server/document-retrival.py

```diff
@@ -23,4 +26,21 @@ def chunk():
     text_splitter = CharacterTextSplitter.from_tiktoken_encoder(
     encoding_name="cl100k_base", chunk_size=CHUNK_SIZE, chunk_overlap=CHUNK_OVERLAP)
     chunks = text_splitter.split_documents(documents)
-    return chunks
\ No newline at end of file
+    return chunks
+
+
+def store_data_in_chromadb():
+    chunk_data = chunk()
+
+    embeddings = HuggingFaceEmbeddings(
+        model_name="sentence-transformers/all-MiniLM-L6-v2"
+    )
+
+    vectorstore = Chroma.from_documents(
+        documents=chunk_data,
+        embedding=embeddings,
+        persist_directory=CHROMA_DIR
+    )
```

Comment:
you forgot to call persist so that it stores the data
```suggestion
    )
    vectorstore.persist()
```

Reply:
oh okay i thought it wasn’t necessary since the chromadb was already kind of doing it

Reply:
i will add it now

Reply:
added